src/contracts/contract_deployer.star

```diff
@@ -1,4 +1,4 @@
-IMAGE = "ethpandaops/optimism-contract-deployer:latest"
```

Comment:
we prob want to ask PandaOps team to rebuild this

Reply:
Would be able to rebuild, once the PR branch ([cl/latest-getting-started](https://github.com/ethereum-optimism/optimism/tree/cl/latest-getting-started)) is merged in.